app/views/partners/_category_filter.html.erb

```diff
@@ -2,38 +2,41 @@
    %>
 
 <div class="breadcrumb__filters filters">
-  <div class="js-filters-toggle filters__toggle">
+  <div class="js-category-filters-toggle filters__toggle">
     <a href="#" data-turbo="false"><span class="icon icon--arrow-down">↓</span> <span class="filters__link">Category</span></a>
   </div>
   
   <%= form_tag partners_path, method: :get, class: 'filters__form', data: { turbo: false } do %>
-  <div class="filters__dropdown filters__dropdown--hidden js-filters-dropdown">
-    <div class="filters__group" >
-      <% filter.categories.each do |category| %>
-      <div class="filters__option">
-        <%= radio_button_tag 'category', category.id, filter.current_category?(category), class: "tag__button" %>
-        <%= label_tag 'category', category.name, class: "tag__label" %>
+    <% if params[:neighbourhood] %>
```

Comment:
I feel that this is breaking the neighbourhood filter abstraction by reading params directly from the query. If the neighbourhood filter is changed this code won't be aware of that.

I don't think there would be anything bad with sending in both filters as they both need to be aware of each other? Then this logic could be moved into the neighbourhood filter where it will all be in one place.